subprojects/docs/src/docs/release/notes.md

```diff
@@ -129,6 +129,10 @@ task myTask {
 In this release we deprecate calling `TaskInternal.execute()`. Calling `task.execute()` should never be necessary.
 There are better ways for re-using task logic, for example by using [task dependencies](userguide/more_about_tasks.html#sec:adding_dependencies_to_tasks), [task rules](userguide/more_about_tasks.html#sec:task_rules), extracting a re-usable piece of logic from your task which can be called on its own (e.g. `project.copy` vs. the `Copy` task) or using the [worker API](userguide/custom_tasks.html#worker_api).
 
+### Other deprecations
+
+* `CompileOptions.bootClasspath` is deprecated in favor of the new `bootstrapClasspath` property.
```

Comment:
Shouldn't we call out the new feature in some place? Or is it too difficult to describe what the advantage is?

Reply:
I think this is not so much a new feature as a fix to a previously badly declared one. I would not call it out more than the deprecation.